### PR TITLE
SLA-bot add ARMSignedOff to ignoredLabels

### PR DESCRIPTION
**Pull request by ruowan**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: .github/sla.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@
         - WaitForARMFeedback
         - DoNotMerge
         - WaitingForAzureAPIBoard
+        - ARMSignedOff
```
Comment (yungezz) [inline .github/sla.yml]: add one more `AzureAPIBoardSignedOff`

File: .github/sla.yml [hunk 2]
```diff
@@ -18,6 +19,7 @@
...
```
Comment (yungezz) [inline .github/sla.yml]: same as above

File: .github/sla.yml (hunk 1 above)
Comment (ruowan) [reply .github/sla.yml]: > add one more AzureAPIBoardSignedOff

done

Comment (yungezz) [reply .github/sla.yml]: second thought on filtering `ARMSignedOff` label, it's ok to filter out those labels in private repo, but not in public repo. After a PR is labelled with "ARMSignedOff`, if no `DoNotMerge` label at same time, the PR owner should be notified to merge the PR as soon as possible. Can we do combination filtering? such as 
if `DoNotMerge` and 'ARMSignedOff`, then no ping;
If 'DoNotMerge` and !`ARMSignedOff`, then ping;

Comment (ruowan) [reply .github/sla.yml]: For complex conditions. I think I can develop a new feature about evaluating label condition expressions. For example: condition: `(DoNotMerge&&!ARMSignedOff)||WaitForARMFeedback`. 

For public and private repo syncing issues. I think this is a special case. I can predefine repository variable and add it in condition sections or other defined sections. For example:  `$REPOSITORY_NAME==Azure/azure-rest-api-specs`

It's a more flexible solution.

Comment (yungezz) [reply .github/sla.yml]: sounds good. Then what's the plan for this PR?

Comment (ruowan) [reply .github/sla.yml]: I suggest we can merge it to master. because it will bother rehe every day during the holiday.

**Comment by yungezz** [comment]
close the PR for now, reopen it when bot ready.

**Pull request closed without merging**